src/amo/components/GuidesAddonCard/index.js

```diff
@@ -0,0 +1,218 @@
+/* @flow */
+import * as React from 'react';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+
+import AddonTitle from 'amo/components/AddonTitle';
+import AddonCompatibilityError from 'amo/components/AddonCompatibilityError';
+import { makeQueryStringWithUTM } from 'amo/utils';
+import AMInstallButton from 'core/components/AMInstallButton';
+import {
+  INCOMPATIBLE_NOT_FIREFOX,
+  INSTALL_SOURCE_GUIDES_PAGE,
+  UNKNOWN,
+} from 'core/constants';
+import { withInstallHelpers } from 'core/installAddon';
+import translate from 'core/i18n/translate';
+import { getAddonByGUID } from 'core/reducers/addons';
+import { getVersionById } from 'core/reducers/versions';
+import { getErrorMessage } from 'core/utils/addons';
+import { getClientCompatibility } from 'core/utils/compatibility';
+import Card from 'ui/components/Card';
+import Icon from 'ui/components/Icon';
+import Button from 'ui/components/Button';
+import Notice from 'ui/components/Notice';
+import type { AddonType } from 'core/types/addons';
+import type { AddonVersionType } from 'core/reducers/versions';
+import type { AppState } from 'amo/store';
+import type { UserAgentInfoType } from 'core/reducers/api';
+import type { I18nType } from 'core/types/i18n';
+import type { ReactRouterLocationType } from 'core/types/router';
+import type { WithInstallHelpersInjectedProps } from 'core/installAddon';
+
+import './styles.scss';
+
+type Props = {
+  addonGuid: string,
+  addonCustomText: string,
+  staffPick?: boolean,
+};
+
+type InternalProps = {
+  ...Props,
+  ...WithInstallHelpersInjectedProps,
+  _getClientCompatibility: typeof getClientCompatibility,
+  addon: AddonType,
+  clientApp: string,
+  currentVersion: AddonVersionType,
+  defaultInstallSource: string,
+  installError: string,
+  installStatus: string,
+  i18n: I18nType,
+  location: ReactRouterLocationType,
+  userAgentInfo: UserAgentInfoType,
+};
+
+export class GuidesAddonCardBase extends React.Component<InternalProps> {
+  static defaultProps = {
+    _getClientCompatibility: getClientCompatibility,
+    staffPick: true,
+  };
+
+  renderInstallError() {
```

Comment:
I thought about this but thought this would be as a follow up PR since that would turn into a dependency.

I guess I can do that fairly quick tho and try to get that in first.

Reply:
You could land this patch with the duplicated code and delete it later in a follow up issue -- that would be fine with me.

Reply:
REF: #6902